src/formatDate.js

```diff
@@ -51,6 +51,52 @@
 
 function formatDate(date, fromFormat, toFormat) {
   // write code here
+  const arr = [];
+  const arrSplit = date.split(fromFormat.pop());
+  const arrToFormat = toFormat.pop();
+  let year = '',
+    month = '',
+    day = '';
```

Comment:
you can just declare without initializing:
```suggestion
  let year, month, day;
```